cmake/test.cmake

```
@@ -0,0 +1,5 @@
+add_executable(test_mat ${PROJECT_SOURCE_DIR}/test/test_mat.c)
+target_link_libraries(test_mat matio)
+
+add_executable(test_snprintf ${PROJECT_SOURCE_DIR}/test/test_snprintf.c)
+target_link_libraries(test_snprintf matio)
```

```suggestion
target_include_directories(test_snprintf
    PUBLIC ${PROJECT_SOURCE_DIR}/src
    PUBLIC ${CMAKE_CURRENT_BINARY_DIR}/src
)
```

test_snprintf does not need to link matio.